### DISCUSSION TITLE
Draft new release from the app.

### DISCUSSION THREAD

**Discussion by alorma** [community/community#140379] [Mobile]
The mobile application (android at least) allows us to modify files and create commits for it.

It will be awesome if "Draft new release" is available on the app. So we can create releases after PRs are reviewed/merged, or when a commit is performed via mobile application.

**Comment by suchakreepanyawong** [comment]
That sounds like a great feature request for the mobile app! Being able to "Draft new release" from the Android GitHub app would streamline the workflow, especially for developers who are on the go and want to create releases right after PRs are merged or commits are performed via the app.

Here’s how you could frame your feature request to submit it to GitHub’s support or issue tracker:

Feature Request: Add "Draft New Release" Option to GitHub Mobile App

Context: The GitHub mobile app currently allows users to modify files, create commits, and manage pull requests. However, the ability to draft and publish new releases is missing. Having this feature in the mobile app would be incredibly useful for developers who work on the go and need to create releases after reviewing/merging PRs or making important commits directly from the app.

Feature Request: Add the functionality to "Draft a New Release" in the GitHub mobile app, so users can:

Create a new release for repositories after a commit is pushed or a pull request is merged.
Add release notes, tag versions, and attach assets (optional) just like on the desktop version.
Use Case:

After merging a PR or committing changes via the mobile app, users can immediately create a release without needing to switch to the desktop site.
This would significantly improve workflow for developers who want to manage their repositories while away from their primary development environment.
Why This Feature Would Be Great:

It allows for a complete mobile experience, from reviewing PRs to making commits and finalizing new releases.
It would streamline workflows, saving time for those who work primarily on mobile or when a desktop isn't readily available.
Thank you for considering this feature!

You can submit this feature request via GitHub’s [support](https://support.github.com/) or open an issue in their public repository if they track features and bugs that way. If you want to contribute directly to this, you could check out GitHub's open-source components, but make sure to see if the mobile app repository is publicly available first.
